app/helpers/password_resets_helper.rb

```diff
@@ -0,0 +1,24 @@
+module PasswordResetsHelper
+  def extract_user_from_reset_params
```

Comment:
`find_user_from_reset_token` I think this is a more communicative name, at all times, let your code tell a Story.

Reply:
ok